### PR TITLE
Update set_up_svs.rst

### PR DESCRIPTION
**Pull request by KwadroNaut**
Fixes updated time settings menu

## Status
Work in progress. Need to add a screenshot.

## Description of Changes

* Tails got some updates, how you got to the settings in the gui has changed.

* Resolves #
https://github.com/freedomofpress/securedrop-docs/issues/428

## Checklist (Optional)

- [ ] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [ ] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

**Comment by KwadroNaut** [comment]
Ready for review.

File: docs/set_up_svs.rst [hunk 1]
```diff
@@ -63,10 +63,9 @@ Key*. This operation requires the Tails administration password to be set
 
 To set the system time:
 
-#. Click the upper right down arrow in the menu bar and select the wrench icon:
-   |select settings|
-#. Select the **Details** section, then click **Date & Time**.
-#. Click **Unlock**. Type in the admin password you set when you
+#. Click in the top left menu Applications, go to **System Tools** and select Settings
```
Comment (nathandyer) [inline docs/set_up_svs.rst:66]: The buttons/targets that users will click should be bolded so they stand out as UI elements in the instructions. There are also some flow issues with the first part of the line. I might suggest a slightly tweaked version like this:

```
#. Click **Applications** in the top left of the screen, then go to **System Tools** and select **Settings**/
```

**Review by nathandyer — CHANGES_REQUESTED** [review]
Thanks @KwadroNaut! I suggested a minor change to one line, but otherwise, it looks good to me. As always, thanks for the PR!

**Comment by eloquence** [comment]
@KwadroNaut Would you like us to pick this one up to get it over the line?

**Comment by nathandyer** [comment]
This PR has been superseded by others since it was originally filed; closing as the original issue is resolved.

**Pull request closed without merging**